tests/hypershift/test_hypershift.py

```diff
@@ -0,0 +1,177 @@
+import logging
+import os
+import shlex
+import shutil
+
+import pytest
+import shortuuid
+from ocm_python_wrapper.cluster import Cluster
+from ocm_python_wrapper.exceptions import ClusterInstallError
+from ocp_utilities.utils import run_command
+from pytest_testconfig import py_config
+from python_terraform import IsNotFlagged, Terraform, TerraformCommandError
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+pytestmark = pytest.mark.hypershift_install
+
+
+def create_vpcs(terraform):
+    LOGGER.info("Create VPCs")
+    try:
+        terraform.plan(dir_or_plan="rosa.plan")
+        terraform.apply(capture_output=False, skip_plan=True, raise_on_error=True)
+    except TerraformCommandError as ex:
+        LOGGER.error(f"Failed to apply Terraform plan, {ex}")
+        raise
+
+
+def destroy_vpcs(terraform):
+    try:
+        LOGGER.info("Destroy VPCs")
+        terraform.destroy(
+            force=IsNotFlagged,
+            auto_approve=True,
+            capture_output=False,
+            raise_on_error=True,
+        )
+    except TerraformCommandError as ex:
+        LOGGER.error(f"Failed to destroy VPCs, {ex}")
+        raise
+
+
+def verify_exported_aws_credentials():
+    missing_envs = []
+    for var_env in ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY"]:
+        if not os.getenv(var_env):
+            missing_envs.append(var_env)
+
+    assert not missing_envs, f"Missing environment variables: {missing_envs}"
```

Comment:
I guess the rosa cmd will [catch](https://github.com/RedHatQE/managed-services-integration-tests/pull/55/files#:~:text=if%20%22ERR%22%20in%20cmd_err%3A) configuration errors, but for AWS do we want to consider check and validate the credentials actual values when session starts? same for ROSA, so any of the configuration fails will be raising before further procedures. 
I'm thinking on local envs when aws credentials need to be set and valid.

Reply:
terraform will fail if the credentials are wrong